subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/testsuites/JUnitJupiterOptionsIntegrationTest.groovy

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.testing.testsuites
+
+import org.gradle.integtests.fixtures.AbstractIntegrationSpec
+import org.gradle.integtests.fixtures.DefaultTestExecutionResult
+import org.gradle.test.fixtures.file.TestFile
+
+class JUnitJupiterOptionsIntegrationTest extends AbstractIntegrationSpec {
```

Comment:
No common base class for these Options tests?  Is this the DAMP tests vs. DRY application code distinction?

Reply:
I tried extracting one. I think they're different enough that it doesn't make sense to extract common build scripts, but there's a little bit of noise that can be hidden in the base class now.

Thanks!

Reply:
DAMP is OK with me for tests like this; for tests I think "easy to debug" outweighs the drawbacks of "verbose and repetitive".